xrpl/deploy-remote-token.js

```diff
@@ -2,25 +2,28 @@
 
 const { Command, Option } = require('commander');
 const { addAmplifierOptions, addChainNameOption } = require('../cosmwasm/cli-utils');
-const { executeTransaction: executeCosmosTransaction } = require('../cosmwasm/utils');
+const { executeTransaction } = require('../cosmwasm/utils');
 const { printInfo, printError } = require('../common');
-const { mainCosmosProcessor, getEvent, getEventAttr } = require('./utils');
+const { mainProcessor } = require('../cosmwasm/processor');
 
 const CONTRACT_CALLED_EVENT_TYPE = 'wasm-contract_called';
 
-const deployRemoteToken = async (config, options, wallet, client, fee) => {
+const deployRemoteToken = async (client, config, options, args, fee) => {
     const { chainName, issuer, currency, tokenName, tokenSymbol, destinationChain } = options;
```

Comment:
We should avoid using a different symbol/name ideally than what's set on the origin chain. Probably should read the on chain value by default, maybe a follow up PR